src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -2,20 +2,89 @@
 
 RSpec.describe Webui::PackageController, vcr: true do
   let(:admin) { create(:admin_user, login: "admin") }
-  let(:user) { create(:user, login: 'tom') }
+  let(:user) { create(:confirmed_user, login: 'tom') }
   let(:source_project) { user.home_project }
   let(:source_package) { create(:package, name: 'my_package', project: source_project) }
   let(:target_project) { create(:project) }
+  let(:package) { create(:package_with_file, name: "package_with_file", project: source_project) }
 
   describe 'submit_request' do
```

Comment:
It was already like that:/ I changed it now. Though this required to update the VCR cassettes. That's why I put it into a new commit